### PR TITLE
Implement end-to-end tDec flow

### PR DESCRIPTION
**Pull request by piotr-roslaniec**
- Related to: https://github.com/nucypher/nucypher-ts/pull/227
- Tested locally
- `StrategyBuilder` has been removed. We use a hard-coded ritual instead.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import App from "./App";
 
 const config: Config = {
   readOnlyChainId: Goerli.chainId,
-  networks: [Goerli]
+  networks: [Goerli],
```
Comment (KPrasch) [inline src/index.tsx]: Mumbai?  What is this network intended to be used for?

**Review by KPrasch — APPROVED** [review]
🧑‍🚀 

I saw it working in today's live demo!

File: src/index.tsx (hunk 1 above)
Comment (piotr-roslaniec) [reply src/index.tsx]: On Mumbai, fixed

File: src/ConditionBuilder.tsx [hunk 2]
```diff
@@ -1,35 +1,30 @@
...
-    parameters: [118],
+  const RpcCondition = {
+    chain: Mumbai.chainId,
+    method: "eth_getBalance",
+    parameters: [":userAddress"],
     returnValueTest: {
-      comparator: '==',
-      value: ':userAddress',
+      comparator: ">",
+      value: "0",
```
Comment (derekpierre) [inline src/ConditionBuilder.tsx:23]: Even though we use a string here, presumably a number is allowed?

Comment (piotr-roslaniec) [reply src/ConditionBuilder.tsx:23]: Yes, both should work fine.

**Pull request merged**